pkg/trace/stats/client_stats_aggregator_test.go

```diff
@@ -290,8 +290,8 @@ func TestCountAggregation(t *testing.T) {
 			c1 := payloadWithCounts(testTime, tc.k, 11, 7, 100)
 			c2 := payloadWithCounts(testTime, tc.k, 27, 2, 300)
 			c3 := payloadWithCounts(testTime, tc.k, 5, 10, 3)
-			kDefault := bucketAggregationKey{}
-			cDefault := payloadWithCounts(testTime, kDefault, 0, 2, 4)
+			keyDefault := bucketAggregationKey{}
```

Comment:
Just out of curiosity - why?

Reply:
`revive` sees `kDefault` as Hungarian notation for defining a constant, which it doesn't accept (see: https://github.com/golang/lint/issues/258, https://github.com/mgechev/revive/blob/5b4bd9a3c2b694d5030b8bc6fa45560fc1b600e1/rule/var-naming.go#L90-L98) and wants it to be replaced by `Default`. That's why it was replaced by the more explicit `keyDefault`, which doesn't trigger this rule.